build/configs/rtl8721csm/loadable_apps/Make.defs

```diff
@@ -211,6 +211,7 @@ else
 	LDELFFLAGS += -T $(TOPDIR)/userspace/userspace_apps.ld
 endif
 
+LDFLAGS += --gc-sections
```

Comment:
Can you check all other configs in Realtek board for removing the unused sections

Reply:
done for all realtek configs.